src/Uno.UWP/ApplicationModel/Background/BackgroundTaskDeferral.Android.cs

```diff
@@ -0,0 +1,24 @@
+﻿
+#if __ANDROID__

```

Comment:
You can remove the `#if` as long as the name of the file ends with `.Android.cs`